react-main/src/app/components/Login/layout.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { reduxForm, Field } from 'redux-form';
+import styles from './styles.scss';
+
+export const minLength = min => value =>
+  value && value.length < min ? ` Must be ${min} characters or more` : undefined
+
+const minLength8 = minLength(8)
+
+const required = value => (value ? undefined : ' Required');
+
+const renderField = ({
```

Comment:
Prefer function components instead of arrow function components:
https://github.com/Wolox/tech-guides/blob/master/react/style-guide.md#class-vs-reactcreateclass-vs-stateless